src/pathToVictory/services/pathToVictory.service.ts

```diff
@@ -352,24 +332,51 @@ export class PathToVictoryService extends createPrismaBase(
         message: candidateSlackMessage + debugMessage,
         channel: SlackChannel.botPathToVictoryIssues,
       })
+      // Mark as Failed
+      await this.completePathToVictory(
+        campaign.slug,
+        pathToVictoryResponse,
+        false,
+        P2VStatus.failed,
+        {
+          officeName,
+          electionLevel,
+          electionState,
+          electionCounty,
+          electionMunicipality,
+          subAreaName,
+          subAreaValue,
+          positionId,
+        },
+      )
+      await this.crmService.handleUpdateCampaign(
+        campaign,
+        'path_to_victory_status',
+        P2VStatus.failed,
+      )
+      return false
     }
-
-    await this.crmService.handleUpdateCampaign(
-      campaign,
-      'path_to_victory_status',
-      'Waiting',
-    )
-    return false
   }
 
   async completePathToVictory(
     slug: string,
     pathToVictoryResponse: {
-      counts: VoterCounts
+      counts: P2VCounts
       electionType: string
       electionLocation: string
     },
     sendEmail = true,
+    p2vStatusOverride?: P2VStatus,
```

Comment:
Instead of slapping this _optional_ arg in the middle of the order, requiring `undefined` to be passed in for it's location in most of the cases where it's called in the app, why not just put it at the _end_ of the order so that it's _truely_ optional?

Or better yet.. change it into an `options` object arg w/ a type definition so that you can just include the key, or omit it?